File: api/GoatSlipsApi/Models/Database/TimeSlip.cs
namespace GoatSlipsApi.Models.Database
{
    public class TimeSlip
    {
        public int Id { get; set; }
        public byte Hours { get; set; }
        public byte Minutes { get; set; }
        public DateTime Date { get; set; }
        public int UserId { get; set; }
        public int ProjectId { get; set; }
        public int? TaskId { get; set; }
        public int? LaborCodeId { get; set; }
    }
}

File: client/src/types/TimeSlip.ts
export interface TimeSlip {
    id: number;
    hours: number;
    minutes: number;
    date: Date;
    userId: number;
    projectId: number;
    taskId: number | null;
    laborCodeId: number | null;
}

export interface ExportableTimeSlip {
    username: string;
    project: string;
    task: string;
    laborCode: string;
    date: string;
    hours: number;
    minutes: number;
}
